services/backend-next/README.md

```diff
@@ -0,0 +1,24 @@
+# [Backend next](https://github.com/SciCatProject/scicat-backend-next)
+
+The SciCat backend-next is a rewrite of the original backend, built on top of the NestJS framework.
+
+## Configuration options
+
+The backend-next service is mainly configured via environment variables. For an extensive list of available options see [here](https://scicatproject.github.io/documentation/Development/v4.x/backend/configuration.html).
```

Comment:
I'd rather put [this](https://github.com/SciCatProject/scicat-backend-next/blob/master/README.md) here